global.php

```diff
@@ -627,7 +627,7 @@
 		$can_access_moderationqueue = false;
 	}
 
-	if($can_access_moderationqueue || ($mybb->user['ismoderator'] && $mybb->usergroup['canmodcp'] == 1 && $mybb->usergroup['canmanagereportedcontent'] == 1))
+	if($can_access_moderationqueue || ($mybb->user['ismoderator'] && $mybb->usergroup['canmodcp'] == 1 && $mybb->usergroup['canmanagereportedcontent'] == 1) && $mybb->settings['reportmethod'] == "db")
```

Comment:
It'd be more optimal at the beginning of https://github.com/dequeues/mybb/blob/fix2378/global.php#L592